tox.ini

```diff
@@ -33,3 +34,11 @@ deps =
     twine
 commands =
     {envpython} -m twine upload {toxinidir}/dist/*
+
+[testenv:type-check]
+skip_install = true
+deps =
+    mypy
+    django-stubs
```

Comment:
We probably need to pin these (both of these packages change a lot), either manually here, or using `pip-tools` which would be a little more complex, but is more future proof.

Reply:
Yes especially the stubs package!